### PR TITLE
feat: add migration to create in app integration

### PR DESCRIPTION
**Pull request by davidsoderberg**
### What change does this PR introduce?

Add migration to create in app integration for existing orgs.

### CODE REVIEW

File: apps/api/migrations/in-app-integration/in-app-integration.migration.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    const environments = await environmentRepository.findOrganizationEnvironments(organization._id);
+    for (const environment of environments) {
+      await integrationRepository.create({
+        _environmentId: environment._id,
+        _organizationId: organization._id,
+        providerId: InAppProviderIdEnum.Novu,
+        channel: ChannelTypeEnum.IN_APP,
+        credentials: encryptCredentials({
+          hmac: environment.widget.notificationCenterEncryption,
```
Comment (scopsy) [inline apps/api/migrations/in-app-integration/in-app-integration.migration.ts:32]: The hmac field is not encrypted right?

Comment (davidsoderberg) [reply apps/api/migrations/in-app-integration/in-app-integration.migration.ts:32]: No

**Review by scopsy — APPROVED** [review]

File: apps/api/migrations/in-app-integration/in-app-integration.migration.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      });
+
+      if (count === 0) {
+        const response = await integrationRepository.create({
+          _environmentId: environment._id,
+          _organizationId: organization._id,
+          providerId: InAppProviderIdEnum.Novu,
+          channel: ChannelTypeEnum.IN_APP,
+          credentials: encryptCredentials({
+            hmac: environment.widget.notificationCenterEncryption,
```
Comment (djabarovgeorge) [inline apps/api/migrations/in-app-integration/in-app-integration.migration.ts:40]: Should we remove the environment.widget.notificationCenterEncryption under the environment? if yes would it be part of this cycle?

Comment (davidsoderberg) [reply apps/api/migrations/in-app-integration/in-app-integration.migration.ts:40]: I think we should, not sure we should do it this cycle. My suggestion is to do it in a later release...

**Pull request merged**